### ISSUE TITLE
Always have the same error when trying to ingest data

### ISSUE THREAD

**Issue by elFerZur** [geodocker/geodocker-geomesa#8]
I install the project into 2 different environments: windows 10 and debian, and always get the same error when trying to ingest data: Zookepper instance is not responding in 5 sg. All the containers are up and running. ¿Where is the problem?

**Comment by elahrvivaz** [comment]
Are you running the ingest from inside the accumulo master docker? You need to do that to pick up the correct environment variables. Alternatively you can specify zookeeper, etc manually on the command line

**Comment by elFerZur** [comment]
Yes, I running the ingest from inside the accumulo master docker. Always
the same error, can't connect with Zookeeper.
I try to put "-z" option, but it doesn't work.

Inside the accumulo master docker, if I use "zookeeper-client", located in
/usr/bin/, fails to connect. But, if I use "zookeper-client -server
172.28.4.180:2181", it connects to zookeeper. But I have to use the IP of
the server where docker is installed, if I use localhost, it doesn't work.

Why doesn't work the -z command line option in geomesa ingest?.

Regards:
F.Zuriaga

2017-10-02 15:33 GMT+02:00 Emilio <notifications@github.com>:

**Comment by FxKu** [comment]
Had the same problem and solved it by editing the `accumulo-site.xml` file in the `/opt/accumulo/conf` folder:
```xml
<property>
  <name>instance.zookeeper.timeout</name>
  <value>15000</value>
  <description>timeout limit for zookeeper</description>
</property>
```

After saving I ran the `accumulo_env.sh` script.
